homework.js

```diff
@@ -0,0 +1,240 @@
+//Create a function that will get the sum of the numbers between 1 and n and return the answer 
+ // should return 15 because 1+2+3+4+5=15
+
+
+ let total = 0;
+ const summation = function (thenumber){
+ 
+  for(var i = 1; i <= thenumber; i++){
+       total += i;
+     }
+     return total;
+ 
+ 
+ }
+ 
+ summation(4);
+ console.log(total);
+ 
+ 
+
+ //Create a program to get the sum of all the even numbers in a group 
+ // should return 6 because 2+4=6
+
+let total = 0;
+const summationEven = function (thenumber){
+
+ for(var i = 1; i <= thenumber; i++){
+
+       if(i % 2 === 0){
+      total += i;
+      
+      }
+    }
+    return total;
+
+
+}
+
+summationEven(5);
+console.log(total);
+
+
+
+//Create a function to get the average of a group of numbers 
+// should return 4
+
+
+let total = 0;
+let averge = 0;
+const AverageOfNumber = function (thenumbers){
```

Comment:
in javascript remember to use camelCasing 

so name your variables as averageOfNumber